src/javascript/app/base/socket_general.js

```diff
@@ -79,6 +80,7 @@ const BinarySocketGeneral = (() => {
                         BinarySocket.send({ get_account_status: 1 });
                         BinarySocket.send({ payout_currencies: 1 });
                         BinarySocket.send({ mt5_login_list: 1 });
+                        SubscriptionManager.subscribe('transaction', { transaction: 1, subscribe: 1 }, () => false);
```

Comment:
We require the subscription manager because in order to push as many `deposit` and `withdrawal` transactions we need to subscribe to the `transaction`-stream right after a client has authorised, but we also need to maintain the functionality in the **Portfolio** page—where the client _again_ tries to subscribe to the `transactions`-stream to update the page in real-time. Without the subscription manager the API would give us a `AlreadySubscribed` error.